docs/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -142,4 +142,12 @@ This option determines whether interactions spans automaticallly get created whe
 
 The default is `false`.
 
+<Note>
+INP samples currently incur no cost to enable at Sentry. Basic samples contain limited information such as the interaction target, latency, and user. You may wish to enrich your INP samples by setting up [Session Replays](/platforms/javascript/session-replay/) and/or setting up [Browser Profiling](/platforms/javascript/profiling/) instrumentation around your interactive elements to gain further insights into your slowest interactions.
+
+Please note that any Session Replays and Browser Profiles used this way will incur their standard cost depending on your plan and are subject to any sampling rates configured.
+
+This feature is in continuous development at Sentry. We plan to improve the experience and reduce friction in future releases.
```

Comment:
Hmm, this line feels a little out of place here to me.

By "this feature", you mean INP right? If so, I'd move this up to be part of the first note about enableINP or alternatively, we could also just delete this line?

If by "this feature" you meant how INP works with session replays & profiling, I'd be more explicit about that rather than calling it a feature